Sources/TuistKit/Commands/Cache/CacheOptions.swift

```diff
@@ -20,4 +25,10 @@ struct CacheOptions: ParsableArguments {
         help: "When passed it caches the targets for simulator and device using xcframeworks."
     )
     var xcframeworks: Bool = false
+
+    @Option(
+        name: .long,
+        help: "Output type of xcframeworks when --xcframeworks is passed (device/simulator)"
+    )
+    var xcframeworksType: CacheXCFrameworkType?
```

Comment:
IIRC ParsableArgument should support a validate method, we should provide it to validate the input is valid (i.e. if xcframeworksType is passed, then xcframeworks must be passed as well)